packages/yarnpkg-pnp/sources/loader/makeApi.ts

```diff
@@ -1,9 +1,12 @@
 import {ppath, Filename}                                                                                                                                                                   from '@yarnpkg/fslib';
 import {FakeFS, NativePath, PortablePath, VirtualFS, npath}                                                                                                                                from '@yarnpkg/fslib';
+import assert                                                                                                                                                                              from 'assert';
```

Comment:
We don't use `assert` at the moment, it should be a sweeping change in another PR